README.md

```diff
@@ -15,18 +15,40 @@ $ make build_image
 ```
 
 ## Deployment
-Copy the sample configuraion file and fill in:
+Should change this part into 3 sections to selecting each section describing how to configure it using the below methods.
```

Comment:
Change it to
"Configuring OpenStack exporter can be done in 3 ways, choose the one that is suitable for your environment."